service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -285,6 +288,11 @@ protected void handleError(T message, Throwable t) {
 
 
     protected void handleChannelMessage(String channel, T message) {
+    	if (channels.get(channel) == null) {
```

Comment:
this should not be ignored.. if we receive message in channel we are not subscribed to it's a big problem